### PR TITLE
code.google.com no longer supported

### PR DESCRIPTION
**Pull request by naoki-k**
Hello,
I came across the following error:

```
 $ go get code.google.com/p/google-api-go-client
package code.google.com/p/google-api-go-client: unable to detect version control system for code.google.com/ path
```

According to the below link, this is happening because Google Code completely shut down as of January 25th.
Thus, I fixed path from `code.google.com` to `google.golang.org`

reference: 
- http://google-opensource.blogspot.jp/2015/03/farewell-to-google-code.html
- https://code.google.com/archive/p/google-api-go-client/

### CODE REVIEW

File: go/README [hunk 1]
```diff
@@ -1,15 +1,15 @@
 Minimum Go version: go 1.1+
 
 To run these code samples, you will need to install the dependent libraries via
-the "go get" command. These code samples require the goauth2 and google-api-go-client
+the "go get" command. These code samples require the oauth2 and youtube
 libraries which can be installed with the following commands:
 
-   go get code.google.com/p/goauth2/oauth
-   go get code.google.com/p/google-api-go-client/youtube/v3
+   go get google.golang.org/api/oauth2/v2
```
Comment (broady) [inline go/README]: golang.org/x/oauth2

Comment (naoki-k) [inline go/README]: Thank you for your comment.
I made a revision. Would you be able to check it?

File: go/oauth.go [hunk 2]
```diff
@@ -13,7 +13,7 @@ import (
 	"path/filepath"
 	"runtime"
 
-	"code.google.com/p/goauth2/oauth"
+	"google.golang.org/p/oauth2/v2"
```
Comment (odeke-em) [inline go/oauth.go]: Same thing here? `golang.org/x/oauth2`

File: go/README [hunk 3]
```diff
@@ -1,15 +1,15 @@
 Minimum Go version: go 1.1+
 
 To run these code samples, you will need to install the dependent libraries via
-the "go get" command. These code samples require the goauth2 and google-api-go-client
+the "go get" command. These code samples require the oauth2 and youtube
 libraries which can be installed with the following commands:
 
-   go get code.google.com/p/goauth2/oauth
-   go get code.google.com/p/google-api-go-client/youtube/v3
+   go get golang.org/x/oauth2/
```
Comment (odeke-em) [inline go/README]: Minor nit: replace oauth2/ with oauth2, so:
```
go get golang.org/x/oauth2
```

**Comment by naoki-k** [comment]
@odeke-em I'm sorry for the delay in my reply. I fixed two points which you commented. Please review again?

**Comment by naoki-k** [comment]
@odeke-em cc: @broady 
Thank you for your comments and I'm sorry for the delay in my reply.
I fixed some codes which you have commented and changed `oauth.go`'s implementation associated with changing from `code.google.com/p/goauth2/oauth` to `golang.org/x/oauth2`.
Then, I referenced #29 to implement the above.
Please review again.

**Pull request closed without merging**